src/components/app/data/utils.test.js

```diff
@@ -0,0 +1,456 @@
+import { LICENSE_STATUS } from '../../enterprise-user-subsidy/data/constants';
+import { ASSIGNMENT_TYPES, POLICY_TYPES } from '../../enterprise-user-subsidy/enterprise-offers/data/constants';
+import { determineLearnerHasContentAssignmentsOnly } from './utils';
+import { emptyRedeemableLearnerCreditPolicies } from './constants';
+
+describe('determineLearnerHasContentAssignmentsOnly', () => {
```

Comment:
[inform] Existing tests copied from the `enterprise-user-subsidy` module. The original `determineLearnerHasContentAssignmentsOnly` remains but will eventually be removed in future commits once its consuming UI components no longer rely on the current data sources.